android/src/ch/opengis/qfield/QFieldUtils.java

```diff
@@ -0,0 +1,122 @@
+package ch.opengis.qfield;
```

Comment:
License headers

Reply:
@m-kuhn , which header style do you want me to use?

Reply:
Coincidentally, none of the .java files (other than the main QFieldActivity.java) have headers.